helm/reana/templates/reana-server.yaml

```diff
@@ -111,7 +111,25 @@ spec:
             value: ""
           - name: GIT_SSL_NO_VERIFY
             value: "true"
+          - name: REANA_EMAIL_SMTP_SERVER
```

Comment:
Missing, otherwise it uses default k8s env var.

```diff
+            {{- range $key, $value := .Values.db_env_config }}
+            - name: {{ $key }}
+              value: {{ $value | quote }}
+            {{- end }}
```